mantidimaging/gui/windows/cor_tilt/presenter.py

```diff
@@ -118,6 +123,19 @@ def do_add_manual_cor_table_row(self):
         idx = self.model.preview_slice_idx
         self.view.add_cor_table_row(idx)
 
+    def do_plot_cor_vs_slice_index(self):
+        if self.model.model.num_points > 1:
+            fig = plt.figure()
+            ax = fig.add_subplot(111)
+
+            ax.plot(self.model.model.slices, self.model.model.cors)
+
+            fit_data = self.model.preview_fit_y_data
+            if fit_data is not None:
+                ax.plot(self.model.model.slices, fit_data)
```

Comment:
Is it worth adding a legend for when the fit curves are plotted? Or maybe consider a diff-style plot as well where the difference is plotted on a separate axis below?
Also, do the axes need labels? It seems odd not having them...

Reply:
Yes, on the larger plot there is really no excuse for lack of axes labels and legend.